### DISCUSSION TITLE
Hiding the Nintendo RVL-CNT-01-UC and only seeing the X-Input controller.

### DISCUSSION THREAD

**Discussion by Tinzo21** [KeyPuncher/WiinUPro#157] [Ideas]
Would it be able all to hide the main WiiU Pro Controller and only register the newly added X-Input controller like in DS4Windows? To prevent issues like the main controller being registered in Dead or Alive 5 and Trackmania, making play either difficult or impossible.

**Comment by DraakjeYoblama** [comment]
DS4Windows uses HidHide, you should be able to use that on its own to manually hide devices.

It's not as elegant, but it's something
